pkg/apply/task/inv_set_task.go

```diff
@@ -49,7 +51,7 @@ func (i *DeleteOrUpdateInvTask) Start(taskContext *taskrunner.TaskContext) {
 	go func() {
 		var err error
 		if i.Destroy && i.destroySuccessful(taskContext) {
-			err = i.deleteInventory()
+			err = i.deleteInventory(taskContext.Context())
```

Comment:
hmm. should we make deleteInventory take a taskContext to be consistent with updateInventory?

Reply:
This was the only one I saw, but please double check that all the other interfaces have consistent use of either the TaskContext or the Context.

Reply:
updateInventory actually uses other fields from the taskContext, which is why it needs it. deleteInventory just needs the `context.Context`. It's such a tiny function it doesn't seem like it matters to me.

Reply:
You know I love consistency, but I agree it's a small enough that it's not a blocker.